ffv1.md

```diff
@@ -797,7 +810,7 @@ Inferred to be 1 if not present.
 
 ### quant_table_index
 
-`quant_table_index` indicates the index to select the quantization table set and the initial states for the slice.
+`quant_table_index` indicates the index to select the quantization table and the initial states for the slice.
```

Comment:
Since `Quantization table indexes` is designed earlier, I think the term should be used in the definition here.